docs/website/docs/contributing/validation-architecture.md

```diff
@@ -0,0 +1,28 @@
+---
+title:  Validation Architecture
+sidebar_position: 10
+---
```

Comment:
Is this doc relevant any more? I don't remember using/following the recommendations. 
If we do want to have it, I think it's more suitable for odo wiki than odo.dev.